test/config/utility.cc

```diff
@@ -1177,6 +1178,32 @@ bool ConfigHelper::setListenerAccessLog(const std::string& filename, absl::strin
   return true;
 }
 
+void ConfigHelper::initializeTlsKeyLog(
+    envoy::extensions::transport_sockets::tls::v3::CommonTlsContext& tls_context,
+    bool keylog_local_filter, bool keylog_remote_filter) {
+  if (!keylog_local_filter && !keylog_local_filter) {
+    return;
+  }
+  auto tls_keylog_path = tls_context.mutable_tls_keylog()->mutable_logfile_path();
+#ifdef _MSC_VER
+  *tls_keylog_path = std::string("%temp%\\keylog");
```

Comment:
For all platforms, use `TestEnvironment::temporaryPath(TestUtility::uniqueFilename());` to get a unique temp file.

Reply:
will fix